src/White.jsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircle } from "@fortawesome/free-solid-svg-icons";
+
+export default function White(props) {
+  const { AllshoesName, AllImage } = props;
+
+  return (
+    <>
+      <div className="container">
+        <div className="listContainer">
```

Comment:
If this is a list then maybe a `<ul />` element would be better (you can style it however you want)